### PR TITLE
first attempt

### PR DESCRIPTION
**Pull request by AndriiBrus**
- [DEMO LINK](https://AndriiBrus.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://AndriiBrus.github.io/layout_moyo-header/report/html_report/)

- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x]  CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] **CSS Variable** is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](AndriiBrus.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](AndriiBrus.github.io/layout_moyo-header/report/html_report/)
```
Comment (Viktor-Kost) [inline readme.md:31]:
```suggestion
- [DEMO LINK](https://AndriiBrus.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://AndriiBrus.github.io/layout_moyo-header/report/html_report/)
```

File: src/index.html [hunk 2]
```diff
@@ -15,8 +15,107 @@
...
+    <header class="header">
+      <a
+        href="logo"
+        class="header__logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="header_logo"
+        />
+      </a>
```
Comment (Viktor-Kost) [inline src/index.html:42]:
```suggestion
      <a
        href="#"
        class="header__logo_link"
      >
        <img
          src="/src/images/logo.png"
          alt="header__logo_img"
        />
      </a>
```

File: src/index.html [hunk 3]
```diff
@@ -15,8 +15,107 @@
...
+      <nav class="header__nav">
+        <ul class="header__nav__ul">
+          <li class="header__nav__ul__li">
+            <a
+              href="Apple"
+              class="header__nav__ul__li__a is-active"
+            >
```
Comment (Viktor-Kost) [inline src/index.html:49]: apply these recommendations to the code below as well
```suggestion
      <nav class="header__nav nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#apple"
              class="nav__link is-active"
            >
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,70 @@
...
+.header {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 50px;
+}
+
+.header__logo {
+  position: absolute;
+  top: 10px;
+  left: 50px;
```
Comment (Viktor-Kost) [inline src/style.css:20]: no needs to make it absolute
```suggestion
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,70 @@
...
+  font-style: normal;
+
+  --text-color: #000;
+  --hover-text-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 50px;
```
Comment (Viktor-Kost) [inline src/style.css:14]:
```suggestion
  padding: 0 50px;
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,70 @@
...
+  position: absolute;
+  top: 10px;
+  left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.header__nav__ul {
+  display: flex;
+  flex-direction: row;
```
Comment (Viktor-Kost) [inline src/style.css:27]: default
```suggestion
```

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,70 @@
...
+  margin-left: 0;
+}
+
+.header__nav__ul__li__a:hover {
+  color: var(--hover-text-color);
+}
+
+.is-active {
+  color: var(--hover-text-color);
+}
```
Comment (Viktor-Kost) [inline src/style.css:60]:
```suggestion
.header__nav__ul__li__a:hover,
.is-active {
  color: var(--hover-text-color);
}
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Review by etojeDenys — CHANGES_REQUESTED** [review]
your links still broken, try to fix them and then re-request me 
image

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,63 @@
-body {
+.body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+
+  --text-color: #000;
+  --hover-text-color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/style.css:8]: Create variables with root selector

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,63 @@
-body {
+.body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
```
Comment (volodymyr-soltys97) [inline src/style.css:4]: Move these styles to the html selector

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,63 @@
...
+  font-style: normal;
```
Comment (volodymyr-soltys97) [inline src/style.css:5]: Default value, so you can just remove it
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Almost done!
Let's make your code better

**Review by lerastarynets — APPROVED** [review]
lgtm 🚀

**Pull request closed without merging**